### DISCUSSION TITLE
Does Faster Whisper work better with WAV or MP3 files?

### DISCUSSION THREAD

**Discussion by toanhuynhnguyen** [SYSTRAN/faster-whisper#1124] [Q&A]
Does Faster Whisper work better with WAV or MP3 files?

**Comment by MahmoudAshraf97** [comment]
No difference as long as the decoded result is the same

**Reply by toanhuynhnguyen** [reply]
But with dual-channel audio MP3 files, I extract channels to MP3 or WAV. Batched Faster Whisper sometimes works better with WAV than MP3 and vice versa (I mean the issue of missing sentences).
The commands to extract channels from dual-channel audio MP3 files to WAV or MP3 files:
- ffmpeg -y -i dualchannel.mp3 -map_channel 0.0.0 channel_0.wav -map_channel 0.0.1 channel_1.wav
- ffmpeg -y -i dualchannel.mp3 -map_channel 0.0.0 channel_0.mp3 -map_channel 0.0.1 channel_1.mp3

**Reply by dgoryeo** [reply]
Hi @toanhuynhnguyen, would you explain a bit more, do you get less "missing sentences" with wav? Thanks.

**Reply by toanhuynhnguyen** [reply]
@dgoryeo occasionally, sometimes "missing sentences" with wav, sometimes "missing sentences" with mp3
